index.html

```diff
@@ -2,8 +2,65 @@
 <html lang="en-GB">
   <head>
     <meta charset="UTF-8" />
-    <link rel="stylesheet" href="styles.css" />
-    <title>Rock, Paper, Scissors</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="wrestlingStyle.css" />
+    <link rel="stylesheet" href="animista.css" />
+    
+    <title>ULTIMATE FIGHTING SCHOOL</title>
   </head>
-  <body></body>
+  <article>   
+  <body class = "text" >
+    <h1 class = "center" id ="mainTitle"> ULTIMATE FIGHTING SCHOOL </h1>
+
+    <div class = "center" id = "name-input">
+      <p class = "center"> <strong>What's your warrior name? &nbsp</strong>
+      <input type="text" id="changeText" value="" maxlength="10" >      
+      <button id = "text-button">Enter name</button>
+      </p>
+      </div>
+
+     <div class = "center" id >  
+       Wrestler smashes Samurai <br>
+       Samurai cuts Sumo <br>
+       Sumo squishes Wrestler!    <br>
+       </p>
+       
+      </div>
+     
+      <h2 class = "center" id = "rules"> <span class ="you"> Click to </span> &nbsp pick your champion </h2> 
+      
+      <strong class = "center" id = "playerChoiceText">Rocky the Wrestler, ˈPāpər the Sumo or Samurai Scissors!</strong>
+             
+       <div class = "center imageContainer"> 
+
+       <img src="Wrestler 1.png"  alt="cartoon wrestler" width="130" height="180" class="center images" id = "rock">
```

Comment:
It's valid to put things like width and height in-line here, but I'd suggest moving them to the CSS file instead; that way, you've separated out your concerns and can reap all the good external style sheet benefits like having a single source of truth so you're able to change width/height on multiple images all at once.